File: uri/uri_python/iniciante/p1973.py
N = int(input())
#entry = input().split(" ")
entry = []
for i in range(999999):
    entry.append('999999')
entry.append('1000000')
for i in range(len(entry)):
    entry[i] = int(entry[i])
soma = sum(entry)
soma_aux = 0
roubadas = 0
for i in range(N):
    if entry[i] == 1:
        soma_aux += 1
    if entry[i] % 2 == 0:
        soma += soma_aux
        roubadas = i
        break
atacadas = i + 1
roubadas += atacadas
print(atacadas, soma - roubadas)

File: uri/uri_cpp/iniciante/p1973.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <stdlib.h>
#include <stdio.h>
#include <math.h>
#include <vector>

using namespace std;

int main() {
   	unsigned long long int i, N, in, soma = 0, soma_aux = 0, roubadas = 0, atacadas = 0;
	bool flag = true;

	cin >> N;
	
	for (i = 0; i < N; i++) {
		cin >> in;
		soma += in;
		if (flag) {
			if (in % 2 == 1) {
				atacadas++;
				roubadas++;
				if (in > 1)
					soma_aux++;
			} else {
				atacadas++;
				roubadas++;
				roubadas += soma_aux;
				flag = false;
			}
		}
	}
	
	cout << atacadas << " " << (soma - roubadas) << endl;
    return 0;
}
